src/App.tsx

```diff
@@ -13,7 +13,7 @@ const Info = styled.p`
   color: white;
   padding: 10px;
   height: 100px;
-  width: 200px;
+  width: 500px;
```

Comment:
I made the Info component wider by default, because as part of the challenge, I'm asking the candidate to write a couple media queries to handle sizing on tablet/mobile.

Reply:
makes sense